Base 1.5. Верстка страницы логина

tUnknownLegend opened the pull request.

Liza1040 committed:
Добавила страницу регистрации

tUnknownLegend committed:
исправления для .form__input

+border-left-with. Так же цвета вынесены в константы

tUnknownLegend committed:
сделана основа для хедера

Нужно привести css в адекватный вид, расположить кнопки равнее, перекрасить svg и добавить лого в hi-res

Liza1040 committed:
Поправила header: выровнила элементы, поправила стили

thelvv committed:
Добавлена страница логина и частично главная, шаблоны распределены по файлам

tUnknownLegend committed:
главная страница готова

так же добавлен цвет фона в логине и регистрации в боковых дивах

tUnknownLegend committed:
добавлены теги header и footer, длинна страницы теперь своя у каждой страницы

tUnknownLegend committed:
исправление ширины для поля поиска

tUnknownLegend committed:
страница логина

Liza1040 committed:
Поправила стили с учетом базовых стилей кнопки, цветов и шрифтов

tUnknownLegend committed:
унификация css классов

merge from Base-1.2_html_make_up

DPeshkoff commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Base-1.5: Сверстать login страницу](https://trello.com/c/ubL15Vv7/10-base-15-%D1%81%D0%B2%D0%B5%D1%80%D1%81%D1%82%D0%B0%D1%82%D1%8C-login-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%83)

DPeshkoff commented on login.html:
```diff
@@ -0,0 +1,102 @@
+                    </div>
+
+                    <input type="submit" class="button-default form__submit text-normal-default" value="Войти">
+                    <div class="line"></div>
```
[А зачем дивак?](https://developer.mozilla.org/ru/docs/Web/HTML/Element/hr)

DPeshkoff commented on login.html:
```diff
@@ -0,0 +1,102 @@
+            </div>
+        </div>
+    </div>
+    <div class="paint-background"></div>
+    <div class="paint-background"></div>
```
Сразу два, страшно

DPeshkoff commented on login.html:
```diff
@@ -0,0 +1,102 @@
+
+</div>
+</body>
+</html>
```
\n

DPeshkoff commented on login.css:
```diff
@@ -0,0 +1,59 @@
+}
+
+.right {
+    background-color: #EEEEEE;
```
такое тоже бы в переменные, используется в `button-login`

DPeshkoff commented on login.css:
```diff
@@ -0,0 +1,59 @@
+}
+
+.button-login:hover {
+    background: var(--green-correct);
```
Зеленый - не ок, к макету тоже писали, слишком акцентирует внимание
Давайте как на странице регистрации

DPeshkoff reviewed (changes requested):
Основное ревью [здесь](https://github.com/frontend-park-mail-ru/2022_2_Base/pull/2), сюда отписал конкретно по странице

tUnknownLegend committed:
убраны пустые дивы, исправлена строка поиска

по возможности выбраны кратные размеры, выравнены и уравнены в размере правые кнопки в хедере

tUnknownLegend committed:
merge from Base-1.2_html_make_up

tUnknownLegend committed:
добавлен скрин верстки
